course4/lecture4_en.md

```diff
@@ -0,0 +1,306 @@
+# Modern Programming Concepts: Tuples, Structures, and Enumerated Types
```

Comment:
```suggestion
# Modern Programming Ideology: Tuples, Structures, and Enumerated Types
```
As suggested by @peter-jerry-ye , the course title should be "Modern Programming Ideology".